### PR TITLE
[MOB-4436] encrypt auth data

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 Jira Ticket(s)

* [MOB-4436](https://iterable.atlassian.net/browse/MOB-4436)

## ✏️ Description

* adds `email` and `userId` into our keychain

### CODE REVIEW

File: swift-sdk/Internal/LocalStorage.swift [hunk 1]
```diff
@@ -94,19 +94,36 @@ struct LocalStorage: LocalStorageProtocol {
```
Comment (tapashmajumder) [inline swift-sdk/Internal/LocalStorage.swift:94]: You need to read and write from keychain here.

Comment (tapashmajumder) [reply swift-sdk/Internal/LocalStorage.swift:94]: Maybe adding some tests here to catch problems like this.

Comment (roninopf) [reply swift-sdk/Internal/LocalStorage.swift:85]: Added a general test to verify presence in the keychain, and the lack of presence in the user defaults.

**Review by tapashmajumder — APPROVED** [review]
+1

**Pull request merged**